bittensor/subtensor.py

```diff
@@ -261,22 +262,31 @@ def __init__(
                 type_registry=bittensor.__type_registry__,
             )
         except ConnectionRefusedError as e:
-            bittensor.logging.error(
-                f"Could not connect to {self.network} network with {self.chain_endpoint} chain endpoint. Exiting..."
+            _logger.error(
+                f"Could not connect to %s network with %s chain endpoint. Exiting...",
+                self.network,
+                self.chain_endpoint,
             )
-            bittensor.logging.info(
-                f"You can check if you have connectivity by runing this command: nc -vz localhost {self.chain_endpoint.split(':')[2]}"
+            _logger.info(
+                "You can check if you have connectivity by running this command: nc -vz localhost %s",
+                self.chain_endpoint.split(":")[2],
```

Comment:
Why the change from f-strings to the older-style %s here?

Reply:
with %s I keep readability and keep 120 without a string break